tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -122,13 +132,22 @@ def test_pvc_delete_and_verify_size_is_returned_to_backend_pool(self):
         used_percentage = pod.run_io_and_verify_mount_point(pod_obj)
         assert used_percentage > '90%', "I/O's didn't run completely"
         used_after_creating_pvc = check_ceph_used_space()
-        logger.info(f"Used after creating pvc {used_after_creating_pvc}")
+        logger.info(f"Used after creating PVC {used_after_creating_pvc}")
         assert used_before_creating_pvc < used_after_creating_pvc
-        pod_obj.delete()
-        pvc_obj.delete()
-        verify_pv_not_exists(pvc_obj, self.cbp_obj.name)
+        rbd_image_id = pvc_obj.image_uuid
+        for resource in pod_obj, pvc_obj:
+            resource.delete()
+            try:
+                resource.ocp.wait_for_delete(resource)
+            except TimeoutError:
+                failed_to_delete.append(resource)
+        if failed_to_delete:
+            raise ResourceLeftoversException(
```

Comment:
ResourceLeftoversException is for the environment check leftovers.
Please use Unexpected behavior exception

Reply:
ack